generators/client/templates/react/src/main/webapp/app/modules/administration/user-management/user-management-detail.tsx.ejs

```diff
@@ -35,6 +35,10 @@ export class UserManagementDetail extends React.Component<IUserManagementDetailP
     this.props.getUser(this.props.match.params.login);
   }
 
+  componentWillUnmount() {
+    this.props.reset();
```

Comment:
Actually the fix should be the other way around. The value needs to be reset during the did mount step of create. Can you try that way

Reply:
I tried it but this does not seem to work

Removing the ``user: action.payload.data`` in the reducer when ``CREATE`` or ``UPDATE`` actions are dispatched is working, or it is also possible to dispatch a reset just after the ``UPDATE`` without removing ``user: action.payload.data`` like the #8338 does, both work

Reply:
ok lets do it in the reducer, the object can be reset on action success